### PR TITLE
feat(assets-icons): new icons and update check-bold

### PR DESCRIPTION
**Pull request by majabnils**
# PR Checklist
https://elvia.atlassian.net/wiki/spaces/TEAMATOM/pages/10427498683/Review+prosess

- [ ] Bumpet package?
- [ ] Updated changelog?
- [ ] Correct date in changelog?

## Describe PR briefly:
https://elvia.atlassian.net/browse/LEGO-3555
https://elvia.atlassian.net/browse/LEGO-3557

### CODE REVIEW

File: packages/elvis-assets-icons/config/icons.config.js [hunk 1]
```diff
@@ -142,6 +142,16 @@ module.exports = [
     terms: ['collapse', 'close'],
     thirdparty: { name: ['streamline'], duplicate: ['arrow_up'] },
   },
+  {
+    name: 'ask_help',
+    terms: ['ask', 'question', 'talk', 'bubble'],
```
Comment (TomSchrier) [inline packages/elvis-assets-icons/config/icons.config.js:147]:
```suggestion
    terms: ['ask', 'question', 'talk', 'bubble', '?'],
```

File: packages/elvis-assets-icons/config/icons.config.js [hunk 2]
```diff
@@ -925,7 +945,12 @@ module.exports = [
   { name: 'undo', terms: ['back', 'regret'] },
   { name: 'unlock', terms: [], thirdparty: { name: ['streamline'] } },
   { name: 'upload', terms: [], thirdparty: { name: ['streamline'] } },
-  { name: 'user_testing-color', terms: [], thirdparty: { name: ['streamline'] } },
+  { name: 'user_testing', terms: ['process'], thirdparty: { name: ['streamline'] } },
```
Comment (TomSchrier) [inline packages/elvis-assets-icons/config/icons.config.js:948]:
```suggestion
  { name: 'user_testing', terms: ['process', 'ux'], thirdparty: { name: ['streamline'] } },
```

File: packages/elvis-assets-icons/config/icons.config.js [hunk 3]
```diff
@@ -925,7 +945,12 @@ module.exports = [
...
+  {
+    name: 'user_testing-color',
+    terms: ['process'],
```
Comment (TomSchrier) [inline packages/elvis-assets-icons/config/icons.config.js:951]:
```suggestion
    terms: ['process', 'ux'],
```

File: packages/elvis-assets-icons/config/icons.config.js [hunk 4]
```diff
@@ -415,6 +425,16 @@ module.exports = [
   },
   { name: 'health_safety_environment', terms: ['HMS'], thirdparty: { name: ['streamline'] } },
   { name: 'heating', terms: ['warm up', 'radiator'] },
+  {
+    name: 'hierarchy',
+    terms: ['process', 'organize'],
```
Comment (TomSchrier) [inline packages/elvis-assets-icons/config/icons.config.js:430]:
```suggestion
    terms: ['process', 'organize', 'structure'],
```

File: packages/elvis-assets-icons/config/icons.config.js [hunk 5]
```diff
@@ -415,6 +425,16 @@ module.exports = [
...
+    thirdparty: { name: ['streamline'] },
+  },
+  {
+    name: 'hierarchy-color',
+    terms: ['process', 'organize'],
```
Comment (TomSchrier) [inline packages/elvis-assets-icons/config/icons.config.js:435]:
```suggestion
    terms: ['process', 'organize', 'structure'],
```

File: packages/elvis-assets-icons/config/icons.config.js [hunk 6]
```diff
@@ -960,6 +985,7 @@ module.exports = [
   },
   { name: 'water_heater', terms: ['hot-water tank', 'warmer'] },
   { name: 'washing_machine', terms: ['washer'] },
+  { name: 'web', terms: [], thirdparty: { name: ['streamline'] } },
```
Comment (TomSchrier) [inline packages/elvis-assets-icons/config/icons.config.js:988]:
```suggestion
  { name: 'web', terms: ['www', 'internet', 'globe', 'world'], thirdparty: { name: ['streamline'] } },
```

File: packages/elvis-assets-icons/config/icons.config.js
Comment (TomSchrier) [inline packages/elvis-assets-icons/config/icons.config.js:1]: Remember to add `more_menu-filled` here and in `icons/svg/src`

File: packages/elvis-assets-icons/CHANGELOG.md
Comment (TrygveUrdahl) [inline packages/elvis-assets-icons/CHANGELOG.md:1]: The elvis-assets-icons package needs to be bumped in the `package.json` 👆

**Review by TrygveUrdahl — APPROVED** [review]
LGTM!

File: packages/elvis/CHANGELOG.json [hunk 7]
```diff
@@ -2,6 +2,24 @@
...
+          ],
+          "pages": [{ "displayName": "Icons", "url": "https://design.elvia.io/tools/icon" }]
+        },
+        {
+          "type": "patch",
+          "changes": ["Updated icon <code>check_bold</code>."],
+          "pages": [{ "displayName": "Icons", "url": "https://design.elvia.io/tools/icon" }]
+        }
+      ]
+    },
```
Comment (TomSchrier) [inline packages/elvis/CHANGELOG.json:22]: Remember to include the changes from https://github.com/3lvia/designsystem/pull/2954 here before merge

**Pull request merged**